### DISCUSSION TITLE
Token átadás

### DISCUSSION THREAD

**Discussion by bodortamas** [nav-gov-hu/eRECEIPT#15] [Kérdések és válaszok]
A szakmai anyagokból, bemutatókból úgy tűnik, hogy a NAV mindenképp QR kóddal átadott vásárlói tokenben gondolkodik. A jogszabályban rögzítve lesz, hogy a token átadása kizárólag QR kóddal történhet, vagy más (párhuzamosan vagy helyettesítő) mód is megvalósítható az ePG-ken?

**Comment by renced42** [comment]
A QR kódban elsődlegesen publikus kulcs illetve egyéb más információ is átadásra kerülhet. A rendelet nem fogja tiltani, hogy ezt más módon is meg lehessen tenni pl NFC-n keresztül, de ez plusz szolgáltatásnak kell lennie, tehát a QR kód kezelése nem hagyható el.
